silence/silence.go

```diff
@@ -843,6 +844,11 @@ func (gd *gossipData) mergeDelta(od *gossipData) *gossipData {
 	defer gd.mtx.Unlock()
 
 	for id, s := range od.data {
+		// Only merge silence if not expired
```

Comment:
Could you expand on this comment a bit? I know I'm going to come back to it in the future and need to refresh myself. Something along the lines of:

>If a gossiped silence is expired, skip it. For a given silence duration exceeding a few minutes, active silences will have already been gossiped. Once the active silence is gossiped, its expiration should happen more or less simultaneously on the different alertmanager nodes. Preventing the gossiping of expired silences allows them to be GC'd, and doesn't affect consistency across the mesh.